resources/assetstore/charts/asset-upload-service/templates/deployment.yaml

```diff
@@ -54,10 +54,7 @@ spec:
                   name: assetstore-minio
                   key: accesskey
             - name: APP_UPLOAD_EXTERNAL_ENDPOINT
-              valueFrom:
-                configMapKeyRef:
-                  name: assetstore-minio-docs-upload
-                  key: APP_EXTERNAL_ENDPOINT
+              value: {{ if .Values.global.minioExternalEndpoint }}{{ .Values.global.minioExternalEndpoint }}{{ else }}'https://minio.{{ .Values.global.ingress.domainName }}'{{ end }}
```

Comment:
If you use global variables in this condition and in various places in asset-store charts, maybe would be better to extract this logic to global values as storeExternalEndpoint?

Reply:
`values.yaml` may contain only values, logic is not supported.